src/caret_analyze/architecture/architecture.py

```diff
@@ -356,6 +356,21 @@ def _verify(nodes: Collection[NodeStruct]) -> None:
 
     def update_message_context(self, node_name: str, context_type: str,
                                subscribe_topic_name: str, publish_topic_name: str) -> None:
+        """
+        Update message_context of node_path in "node_name" node.
+
+        Parameters
+        ----------
+        node_name : str
+            name of target node
+        context_type : str
+            type name of updating message_context
```

Comment:
```suggestion
            type name of message_context to be added
```

Reply:
Fix it in [6cf0285](https://github.com/tier4/CARET_analyze/pull/300/commits/6cf0285fa738613d5ed20fe836eed60ce682205c).